schemaless/create_schemaless.py

```diff
@@ -157,24 +143,36 @@ def run(out_file,
                 (OEWDPermits, oewd_permits_file)]:
             if arg:
                 sources.append(source(arg))
-            elif source.DATA_SF_DOWNLOAD and not no_download:
-                dl_sources[executor.submit(get, destdir, source)] = source
+            elif source.DATA_SF_VIEW_ID and not no_download:
+                dest = os.path.join(destdir, "%s.csv" % source.NAME)
+                dl_sources[executor.submit(
+                    download, client, source.DATA_SF_VIEW_ID, dest)] = source
             else:
                 logger.warning("Skipping %s" % source.NAME)
 
+        if not parcel_data_file:
+            parcel_data_file_future = executor.submit(
+                download,
+                client,
+                PARCELS_DATA_SF_VIEW_ID,
+                os.path.join(destdir, 'parcels.csv'))
+
         for future in futures.as_completed(dl_sources):
             try:
                 src = dl_sources[future]
                 sources.append(src(future.result()))
             except Exception:
                 logger.exception("Error downloading data for %s", src.NAME)
                 raise
+        parcel_data_file = parcel_data_file_future.result()
```

Comment:
Does this line handle if a parcel data file is passed in ok?

Reply:
nope, I screwed that up.

Reply:
fixed